extensions/mentions/src/Formatter/UnparseGroupMentions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Mentions\Formatter;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class UnparseGroupMentions
```

Comment:
We don't need the unparsing logic if we only have a single format which is `@"name"#gID`, the unparsing logic was added for usernames to convert `@username` to `@"DisplayName"#uID`